dotcom-rendering/src/components/marketing/epics/ContributionsEpicButtons.tsx

```diff
@@ -186,13 +189,28 @@ export const ContributionsEpicButtons = ({
 	const getChoiceCardCta = (cta: Cta): Cta => {
 		if (
 			showThreeTierChoiceCards &&
-			threeTierChoiceCardSelectedAmount != undefined
+			variantOfChoiceCard &&
+			[
+				'V1_THREE_TIER_CHOICE_CARDS',
+				'V2_THREE_TIER_CHOICE_CARDS',
+			].includes(variantOfChoiceCard) &&
+			!isUndefined(threeTierChoiceCardSelectedAmount)
 		) {
+			if (threeTierChoiceCardSelectedAmount === 0) {
+				return {
+					text: cta.text,
+					baseUrl: addChoiceCardsParams(
+						'https://support.theguardian.com/contribute/checkout?selected-contribution-type=one_off',
+						'ONE_OFF',
+						threeTierChoiceCardSelectedAmount,
+					),
+				};
+			}
 			return {
 				text: cta.text,
 				baseUrl: addChoiceCardsParams(
-					cta.baseUrl,
-					'MONTHLY', // only doing monthly in the first test
+					'https://support.theguardian.com/uk/contribute/checkout',
```

Comment:
should be `https://support.theguardian.com/contribute/checkout` - without the region